contracts/Moloch.sol

```diff
@@ -392,7 +394,7 @@ contract Moloch {
         }
 
         // Make sure there is enough tokens for payments, or auto-fail
-        if (proposal.paymentRequested >= proposal.paymentToken.balanceOf(address(guildBank))) {
+        if (proposal.paymentRequested > proposal.paymentToken.balanceOf(address(guildBank))) {
```

Comment:
we had a test with zero in the bank and zero payment requested in the proposal. The `>=` meant it always set `didPass` to false in this scenario and therefore didn't issues shares etc